polaris-react/src/components/CustomProperties/CustomProperties.tsx

```diff
@@ -43,7 +43,7 @@ export function CustomProperties(props: CustomPropertiesProps) {
 
   return (
     <Component
-      p-color-scheme={colorScheme}
+      data-p-color-scheme={colorScheme}
```

Comment:
This would be a breaking change as overriding custom properties with an attribute selector is the recommended approach for upgrading from the `ThemeProvider` to `CustomProperties` component in the [v8 to v9 migration guide](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v8-to-v9.md#replacing-custom-properties-from-theme-overrides).